server/the-usher-openapi-spec.yaml

```diff
@@ -1061,6 +1097,39 @@ paths:
   /clients/{client_id}/permissions:
     parameters:
     - $ref: '#/components/parameters/clientIdPathParam'
+    get:
+      'x-swagger-router-controller': 'clients/permissions'
+      operationId: getClientPermissions
+      summary: Get a list of permissions for a client
+      tags:
+        - Client Admin APIs
+      security:
+        - bearerAdminAuth: []
+        - bearerClientAdminAuth: []
+      responses:
+        200:
+          description: List of permissions for a client
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  allOf:
+                    - $ref: '#/components/schemas/PermissionObject'
+                    - type: object
+                      properties:
+                        client_id: 
+                          type: string
```

Comment:
@mikelax let me know if you prefer to include `client_id` in the permission object.

Reply:
@k1ch intjibk what you did here is fine. 
This way if we have to reuse the perms object elsewhere it can be (ie. For create operations).